File: retroravelry/src/main/java/com/caseykulm/retroravelry/responses/library/LibraryResponse.java
package com.caseykulm.retroravelry.responses.library;

import com.caseykulm.retroravelry.models.Paginator;
import com.caseykulm.retroravelry.models.volume.ListVolume;
import com.google.auto.value.AutoValue;
import java.util.List;

@AutoValue public abstract class LibraryResponse {

  public abstract Paginator paginator();
  public abstract List<ListVolume> volumes();

}

File: retroravelry/src/main/java/com/caseykulm/retroravelry/responses/library/LibraryResponse.kt
package com.caseykulm.retroravelry.responses.library

import com.caseykulm.retroravelry.models.Paginator
import com.caseykulm.retroravelry.models.volume.ListVolume

data class LibraryResponse(
    val paginator: Paginator?,
    val volumes: List<ListVolume>?
)
